ra/ra_test.go

```diff
@@ -911,9 +909,19 @@ func TestPerformValidationAlreadyValid(t *testing.T) {
 	authzPB, err := bgrpc.AuthzToPB(authz)
 	test.AssertNotError(t, err, "bgrpc.AuthzToPB failed")
 
-	va.RecordsReturn = []core.ValidationRecord{
-		{Hostname: "example.com"}}
-	va.ProblemReturn = nil
+	fakeHostname := "example.com"
+	fakePort := "8080"
+	va.ResultReturn = vapb.ValidationResult{
+		Records: []*corepb.ValidationRecord{
+			{
+				AddressUsed: []byte("192.168.0.1"),
```

Comment:
Yes, this is because I've updated the VA's request/response types, but not the RA's. So previously, the RA just took whatever result it got back from the VA and passed it through directly into its own result without examining it (`ra.go:1624` before this diff). Now, because its own interface types haven't been updated, the RA has to pass the VA's ValidationRecord through `bgrpc.PBToValidationRecord` (`ra.go:1650` after this diff), which rightly complains if these fields aren't set.